packages/sender/src/lib/sender.ts

```diff
@@ -278,6 +278,20 @@ const Sender: WalletBehaviourFactory<InjectedWallet> = async ({
           return res.response;
         });
     },
+
+    async importAccountsInSecureContext({ accounts }) {
+      if (window.near && window.near.isSender) {
+        window.near.batchImport({
+          keystore: JSON.stringify(accounts),
+          network: options.network.networkId,
+        });
+      } else {
+        window.open(
```

Comment:
Just to inform you each module has an [available](https://github.com/near/wallet-selector/blob/main/packages/sender/src/lib/sender.ts#L312) property in the `metadata` object in case of Injected wallets we call the `isInstalled()` function which checks if an extension is installed during setup and sets the value for the `available` property.

Then in UI, you can check it this way: https://github.com/near/wallet-selector/blob/main/packages/modal-ui/src/lib/components/Modal.tsx#L124-L130

Reply:
I think the modal ui already handles uninstalled wallets nicely on it's own way. 
<img width="827" alt="image" src="https://user-images.githubusercontent.com/6027014/205726421-a04861ee-9cec-41d5-92aa-379f0c01431b.png">
I'm implementing on top of modal-ui and also have layer of logic to check for interface implemented. If not, will show this:
<img width="524" alt="image" src="https://user-images.githubusercontent.com/6027014/205727805-24092fb7-6981-4de0-9a64-d04806a9faab.png">

I'm happy to remove `else` part of the logic.

Reply:
Perfect, thank you @kujtimprenkuSQA and @hcho112 🎉